plugins/com.google.cloud.tools.eclipse.sdk.ui/src/com/google/cloud/tools/eclipse/sdk/ui/DefaultOutputLineListener.java

```diff
@@ -10,16 +10,16 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  *******************************************************************************/
-package com.google.cloud.tools.eclipse.appengine.localserver.server;
+package com.google.cloud.tools.eclipse.sdk.ui;
 
 import org.eclipse.ui.console.MessageConsoleStream;
 
 import com.google.cloud.tools.appengine.cloudsdk.process.ProcessOutputLineListener;
 
-public class LocalAppEngineOutputLineListener implements ProcessOutputLineListener {
+public class DefaultOutputLineListener implements ProcessOutputLineListener {
```

Comment:
This name doesn't convey what it does.  `OutputStreamLineWriter`?  `MessageConsoleLineWriter`?